<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectWindow</class>
 <widget class="QWidget" name="selectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="chartsWebView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="content">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>619</width>
      <height>419</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="title1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Panaderias</string>
   </property>
  </widget>
  <widget class="QWidget" name="formWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="bakeriesView">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="noAvailableBakeries">
      <property name="text">
       <string>No hay panaderias disponibles</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
